controllers/admin/admin_mobile_controller.py

```diff
@@ -23,17 +24,46 @@ def get(self):
         ios = all_clients.filter(MobileClient.client_type == ClientType.OS_IOS).count()
         webhook = all_clients.filter(MobileClient.client_type == ClientType.WEBHOOK).count()
 
+        var = Sitevar.get_by_id('notifications.enable')
+        if var is None or not var.values_json == "true":
+            push_enabled = "disabled"
+            disp_enable = ""
+            disp_disable = "disabled"
+        else:
+            push_enabled = "enabled"
+            disp_enable = "disabled"
+            disp_disable = ""
+
         self.template_values.update({
             'mobile_users': all_clients.count(),
             'android_users': android,
             'ios_users': ios,
             'webhooks': webhook,
             'broadcast_success': self.request.get('broadcast_success'),
+            'push_enabled': push_enabled,
+            'disp_enable': disp_enable,
+            'disp_disable': disp_disable,
```

Comment:
These three can be combined into one boolean that gets passed to the template, and the logic should reside in the template.